packages/cli/src/lib/system/docker.ts

```diff
@@ -17,14 +17,25 @@ export function getDockerFileLock(): FileLock {
   return new FileLock(__dirname + "/DOCKER_LOCK", print.error);
 }
 
+export async function isDockerBuildxInstalled(): Promise<boolean> {
+  const { stdout: version } = await runCommand("docker buildx version");
```

Comment:
I tried this on my local machine, and it looks like it doesn't work. What are the setup steps to add buildx support to the docker CLI?

If it requires a certain version of docker to be installed, it'd be nice to add  some nice error messaging for the user so they know that their buildkit options are not being used due to their docker version being too old.

Reply:
unfortunately, it's not that easy and depends on the OS and the way you installed it. It comes included with the docker desktop but if you have installed docker using other means, you need to install buildkit separately.

here's the guide:
https://docs.docker.com/buildx/working-with-buildx/